x-pack/plugins/endpoint/common/types.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export class EndpointAppConstants {
```

Comment:
Did you mean to move a `Class` to this `types.ts` file? I don't see the FE needing it

Reply:
@nnamdifrankie see my prior comment here. I don't think you should have a Class in this file (maybe you meant `interface`?

Reply:
I think @nnamdifrankie is defining constants here, not types necessarily. So I would say that maybe we could move this somewhere else, if we're trying to have this file contain and export types for the app.